lib/samson/process_utils.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Samson
+  module ProcessUtils
+    ATTRIBUTES = ['pid', 'ppid', 'gid', 'user', 'start', 'args'].freeze
+
+    class Process
```

Comment:
smells like unnecessary abstraction ... lots of magic to set and then call it ... might as well do it inline ?